apps/test/solutions/applab/cspunit3.js

```diff
@@ -117,50 +84,29 @@ module.exports = {
         return levelDefinition;
       },
       runBeforeClick: function (assert) {
-        // Paramaterless moveForward. Moves 25 pixels by default
-        var moveForward = 0;
-        var turnLeft = 1;
-
-        var nextIndex = 0;
-        function appendToolboxBlock(blockIndex) {
-          dragToolboxBlock(blockIndex, nextIndex++);
-        }
-
-        appendToolboxBlock(moveForward);
-        assert.equal(Applab.getCode(), 'moveForward();\n');
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(moveForward);
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(moveForward);
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(turnLeft);
-        appendToolboxBlock(moveForward);
+        var expectedCode = '' +
+          'moveForward();\n' +
+          'turnLeft();\n' +
+          'turnLeft();\n' +
+          'turnLeft();\n' +
+          'moveForward();\n' +
+          'turnLeft();\n' +
+          'turnLeft();\n' +
+          'turnLeft();\n' +
+          'moveForward();\n' +
+          'turnLeft();\n' +
+          'turnLeft();\n' +
+          'turnLeft();\n' +
+          'moveForward();\n';
+
+        $("#show-code-header").click();
+        testUtils.setAceText(expectedCode);
+
+        var actualCode = Applab.getCode();
+        assert.equal(actualCode, expectedCode, 'code set properly');
```

Comment:
This test started failing again, because phantom ends up laying things out differently and our attempts to simulate dragging blocks stopped working. I spent an hour or two trying to get it to work again, and ultimately decided that given the difficulty in doing so combined with the flakiness we saw in differences between circle-ci and dev environments, this isn't a very good test. Instead, I just set the editor text directly.